src/relay/transforms/fake_quantization_to_integer.cc

```diff
@@ -270,8 +293,233 @@ class FakeQuantizationRewriter : public MixedModeMutator {
   const bool hard_fail_;
 };
 
+bool is_op_enabled_for_optional_fq2i(const CallNode* call_node) {
```

Comment:
Could we add some comments and advice about what ops should be included in this list?

Reply:
I added a comment of how I selected these operations.